### PR TITLE
Add package.json and wrap code in IIFE

### PR DESCRIPTION
**Pull request by TimonVS**
Add `package.json` so we can install the package with NPM.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "uploadcare-redactor",
+  "version": "1.2.0",
+  "description": "This is Uploadcare plugin for [Imperavi Redactor] text editor. It will allow your users to upload files and images from local device, social networks, cloud storages without any backend code that is usually required to handle uploads.",
+  "main": "uploadcare.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TimonVS/uploadcare-redactor.git"
```
Comment (dmitry-mukhin) [inline package.json]: probably should be updated?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "author": "Uploadcare",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/uploadcare/uploadcare-redactor/issues"
```
Comment (dmitry-mukhin) [inline package.json]: can we have several urls here?
so users could file redactor related issues here and widget related issues [there](https://github.com/uploadcare/uploadcare-widget/issues/)

Comment (TimonVS) [inline package.json]: @dmitry-mukhin The docs specify that you can only have one url there: https://docs.npmjs.com/files/package.json.

**Comment by TimonVS** [comment]
Please don't merge yet. I have to fix the indentation of the code.

**Comment by TimonVS** [comment]
Fixed the identation. It's ready to merge :).

File: package.json (hunk 2 above)
Comment (dmitry-mukhin) [inline package.json]: alrighty!

**Comment by dmitry-mukhin** [comment]
thanks!

**Pull request merged**